### PR TITLE
staging: Add r-sanssouci

### PR DESCRIPTION
**Pull request by mschilli87**
Another R-library I tried to package properly.This one is not only not on CRAN, but also not on Bioconductor (yet). Also, I had to fix the latest commit as a version 0 'release' since there I could not find any official release archive.

### CODE REVIEW

**Comment by mschilli87** [comment]
rebased on latest master

File: bimsb/packages/staging.scm [hunk 1]
```diff
@@ -2665,3 +2665,32 @@ to estimate cell type proportions in bulk RNA-seq data.")
 algorithm} (DSA) for extracting cell-type specific gene expression profiles
 from mixed tissue samples.")
       (license license:gpl2))))
+
+(define-public r-sanssouci
+  (let ((commit "5fe20a9aaf4ac637fa83d9cc73ff1c22de97ca6f")
+        (revision "1"))
+    (package
+      (name "r-sanssouci")
+      (version (string-append "0-" revision "." (string-take commit 9)))
```
Comment (rekado) [inline bimsb/packages/staging.scm]: You can use `git-version` in these cases, e.g. `(version (git-version "0" revision commit))`

File: bimsb/packages/staging.scm [hunk 2]
```diff
@@ -2665,3 +2665,32 @@ to estimate cell type proportions in bulk RNA-seq data.")
 algorithm} (DSA) for extracting cell-type specific gene expression profiles
 from mixed tissue samples.")
       (license license:gpl2))))
+
+(define-public r-sanssouci
+  (let ((commit "5fe20a9aaf4ac637fa83d9cc73ff1c22de97ca6f")
```
Comment (rekado) [inline bimsb/packages/staging.scm:2672]: I like to add a comment here that explains quickly why we use this commit.

File: bimsb/packages/staging.scm (hunk 1 above)
Comment (mschilli87) [reply bimsb/packages/staging.scm]: Thank you. Fixed.

File: bimsb/packages/staging.scm (hunk 2 above)
Comment (mschilli87) [reply bimsb/packages/staging.scm:2672]: Done. :-)

**Comment by rekado** [comment]
Pushed to Guix `master` branch with commit cf2251b9f7a53a3303ea71d197bef4a96997a4c3.  Thanks!

**Pull request closed without merging**